### PR TITLE
More keyboard shortcuts

### LINKED ISSUE #1277: [Flutter] Add keyboard shortcuts for "Run" and "Format".
These two actions are done a lot when writing Flutter code. It would be nice to have shortcuts for these two things.

On MacOS, I'm instinctively expecting "Command + S" and "Command + Shift + F" respectively, although that might just be my personal instincts.

### PR DESCRIPTION
**Pull request by johnpryan**
- adds a format shortcut (cmd + shift +f)
- adds a keymap dialog to the embeds

closes #1277

### CODE REVIEW

File: lib/experimental/keymap.dart [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// HTML for keyboard shortcuts dialog
```
Comment (RedBrogdon) [inline lib/experimental/keymap.dart]: nit: copyright header.

File: lib/experimental/keymap.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import 'dart:html';
+
+import 'package:dart_pad/core/keys.dart';
+
+String keyMapToHtml(Map<Action, Set<String>> keyMap) {
+  DListElement dl = DListElement();
+  keyMap.forEach((Action action, Set<String> keys) {
+    if (!action.hidden) {
+      String string = '';
```
Comment (RedBrogdon) [inline lib/experimental/keymap.dart]: Unlikely to make a noticeable performance difference, but this seems like a good place to use a StringBuffer.

File: lib/experimental/new_embed.dart [hunk 3]
```diff
@@ -528,6 +529,12 @@ major browsers, such as Firefox, Edge (dev channel), or Chrome.
     }, 'Quick fix');
 
     keys.bind(['ctrl-enter', 'macctrl-enter'], _handleExecute, 'Run');
+    keys.bind(['shift-ctrl-/', 'shift-macctrl-/'], () {
+      _showKeyboardDialog();
+    }, 'Shortcuts');
```
Comment (RedBrogdon) [inline lib/experimental/new_embed.dart]: nit: "Show Keyboard Shortcuts"?

File: lib/experimental/keymap.dart (hunk 2 above)
Comment (johnpryan) [reply lib/experimental/keymap.dart]: this code is just moving from lib/new_playground.dart, I'm not sure a StringBuffer is necessary

**Comment by johnpryan** [comment]
I'm going to leave the "or" out for now, since this change was more focused on adding additional shortcuts, not updating the dialogs

**Pull request merged**